app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
+    target_file = parts[2]
+
+    if source_file == target_file:
+        return
+
+    if not os.path.exists(source_file):
+        return
+
+    with (
```

Comment:
You can use shutil.copyfile to do this in one method, it would be like this "shutil.copyfile(source_file_name, target_file_name)"